dataprofiler/tests/profilers/test_histogram_option.py

```diff
@@ -11,7 +11,6 @@ class TestHistogramOption(TestBooleanOption):
     def test_init(self):
         option = self.get_options()
         self.assertTrue(option.is_enabled)
-        self.assertIsNone(option.method)
```

Comment:
Instead of making this `self.assertIsNone` can we instead make it `self.assertEqual(['auto'], option.method)` to show new functionality